spec/system/task_spec.rb

```diff
@@ -11,6 +11,7 @@
     #作成したタスクオブジェクトを各テストケースで呼び出せるようにインスタンス変数に代入
     @user = FactoryBot.create(:user)
     @task = FactoryBot.create(:task)
+    @new_task = FactoryBot.create(:new_task)
```

Comment:
設定ファイルに追記を行うことで、下記のように記載を省略することができます。
```suggestion
    @new_task = create(:new_task)
```